backend/src/main/kotlin/cz/loono/backend/data/model/Account.kt

```diff
@@ -0,0 +1,101 @@
+package cz.loono.backend.data.model
+
+import org.hibernate.Hibernate
+import java.time.LocalDate
+import java.util.Objects
+import javax.persistence.Column
+import javax.persistence.Embeddable
+import javax.persistence.Embedded
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "\"account\"")
+data class Account(
+    @Id
+    @Column(nullable = false)
```

Comment:
Please define all String types as the TEXT data types. There are many advantages.